### ISSUE TITLE
Payments - View VA payment history page error state has no H1

### ISSUE THREAD

**Issue by bacitracin** [department-of-veterans-affairs/va.gov-team#108005]
## Issue
There is no H1 on the View payments page when there is an error / there are no payments

New Cypress Axe rules for headings are coming and will be put into effect April 30. [Slack thread](https://dsva.slack.com/archives/C03R5SBELQM/p1742920275472879) The link to the Axe report expired, but I noticed this one while running an Axe Devtools scan.

Page: https://staging.va.gov/va-payment-history/payments/

<img width="1506" alt="Image" src="https://github.com/user-attachments/assets/4c2c8756-3179-4cb0-98b5-2757dd66bf4d" />
